spec/app.spec.js

```diff
@@ -38,10 +38,32 @@ describe('Test NeutralinoApp Class', function () {
 
     process.stdout.write = originalWrite;
 
-    assert.ok(output.includes('--load-dir-res --path=. --export-auth-info --neu-dev-extension  --url=/ --window-width=500 --window-height=500'));
+    assert.ok(output.includes('--load-dir-res --export-auth-info --neu-dev-extension  --path=./ --window-width=500 --window-height=500 --window-hidden=true'));
```

Comment:
Could you please double-check `--path=./` I think we don't need the trailing slash. Thanks